docs/Integrating Your HMI/Developing the HMI UI/index.md

```diff
@@ -120,6 +120,10 @@ The following graphic shows what should happen when the HMI receives a `UI.SetMe
 
 ## Implementing Soft Buttons
 
+!!! NOTE
+As of Core 8.0, `OnButtonEvent` and `OnButtonPress` notifications will only be forwarded to mobile if the HMI includes an element with button name `CUSTOM_BUTTON` in it's response to `Buttons.GetCapabilities`.
```

Comment:
I would say something more along the lines of:

As of Core 8.0.0 it is important to include CUSTOM_BUTTON in the response to `Buttons.GetCapabilities` to allow an app to subscribe to custom soft buttons.